### ISSUE TITLE
Can't install microG and YT Vanced in Root and NonRoot

### ISSUE THREAD

**Issue by Johnnii360** [YTVanced/VancedManager#605]
**Please only open the issue if the following is true**
- This is an issue in the Vanced Manager and ONLY Vanced Manager (NOT Youtube Vanced/Music/microG)

**Tablet Specifications:**
- Brand: Samsung Galaxy Tab 4 (10.1")
- Operating System: Android (Samsung Stock, rooted)
- Android Version: 5.0.2 (Build LRX22G.T530XXS1BRH1)
- Vanced Manager Version: 2.6.0

**Please describe the problem you are having in as much detail as possible:**
I can't install microG and Youtube Vanced in NonRoot and Root Mode. When I try to install it in Root Mode I get an error. When I try to install it in NonRoot Mode the file will be downloaded and in the installation process the Vanced Manager dialog. is always shown without any Android App Installation Dialog. Rights to install from unknown sources are granted. There are no additional security apps installed.

**Further details:**
Here the error from Root Mode installation:

![20210702_135735](https://user-images.githubusercontent.com/15824634/124275475-de5ea080-db42-11eb-818f-7f8aef8ab28b.jpg)

The Root Mode log: [vancedManagerRoot.logcat.log](https://github.com/YTVanced/VancedManager/files/6754585/vancedManagerRoot.logcat.log)
And here NonRoot: [vancedManagerNonRoot.logcat.log](https://github.com/YTVanced/VancedManager/files/6754587/vancedManagerNonRoot.logcat.log)

**Comment by X1nto** [comment]
Nonroot log shows that manager issued a request dialog but no callback was provided, possibly something is wrong with your ROM. For root you must be using Magisk because other root solutions are not supported.

**Comment by Johnnii360** [comment]
> Nonroot log shows that manager issued a request dialog but no callback was provided, possibly something is wrong with your ROM. For root you must be using Magisk because other root solutions are not supported.

Hmm... is there some additional tests to check what's wrong with my ROM? Maybe I disabled a service with the service disabler?

**Comment by X1nto** [comment]
> > Nonroot log shows that manager issued a request dialog but no callback was provided, possibly something is wrong with your ROM. For root you must be using Magisk because other root solutions are not supported.
> 
> Hmm... is there some additional tests to check what's wrong with my ROM? Maybe I disabled a service with the service disabler?

I think if the service was disabled, it wouldn't log "requesting user confirmation for installation". have you tried restarting?

**Comment by Johnnii360** [comment]
> I think if the service was disabled, it wouldn't log "requesting user confirmation for installation". have you tried restarting?

Yes, many times. It's really weird because it worked on my Samsung Galaxy Note 10.1 2014 Edition Tablet (Android 5.1.x) and Samsung Galaxy Note 8 Smartphone (Android 9) in NonRoot Mode. I really don't know why it's not work on the Tablet of my Mother. If I could avoid a ROM re-install would be very nice.
